src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private CircleSupplier circle = new CircleSupplier();
+    private IsoscelesTrapezoidSupplier isoscelesTrapezoid = new IsoscelesTrapezoidSupplier();
+    private RectangleSupplier rectangle = new RectangleSupplier();
+    private SquareSupplier square = new SquareSupplier();
+    private RectangleSupplier rightTriangle = new RectangleSupplier();

```

Comment:
It seems you've made a mistake by instantiating `RectangleSupplier` for the `rightTriangle` instead of `RightTriangleSupplier`. Make sure that you are using the correct supplier for each figure.